### DISCUSSION TITLE
ZSDR-850 (Trust) shows up as HS1SA but no battery percentage

### DISCUSSION THREAD

**Discussion by dlarge51** [Koenkk/zigbee2mqtt#6033] [Devices]
Hi, I got a ZSDR-850, Brand: Trust. 
I can pair it, and if I hit the test button, it's shows up in Domoticz (light switch shows on for a second and turns to off again). 
But, I don't have battery percentage. It shows 0%. The voltage shows 0 V as well.
Link quality seems to work well. 
Any ideas?

**Comment by EricV-hue** [comment]
Hi,

I also got a ZSDR-850, brand KlikAanKlikUit.
Immediately recognised as a Heiman HS3SA. 
Works as a charm, nice!

**Comment by AKL-1707** [comment]
Hi,

 in July 2026 I tested the Trust ZSDR-850 with Zigbee2MQTT 2.7.2 and a SLZB-06 coordinator. (firmware V3.3.1)

The detector still exposes battery but does not report battery percentage. 

Reading genPowerCfg reporting configuration fails and battery remains unavailable. 

Is this expected for the Trust firmware or is the expose incorrect?”
